README.md

```diff
@@ -595,11 +595,12 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Femi Oluwadamilola](https://github.com/femiOluwadamilola)
   - [Fidal Mathew](https://github.com/FidalMathew)
   - [FitriRibbit](https://github.com/FitriRibbit)
+  - [Fiyinfoluwa Fatuase](https://github.com/Fiiyinfoluwa)
   - [Fran Pérez](https://github.com/franpersanchez)
   - [Franklin Ohaegbulam](https://github.com/frankiefab100)
   - [Frazier Odhiambo](https://github.com/frazie)
   - [Furqan Abid](https://github.com/heyyfurqan)
-  - [Fiyinfoluwa Fatuase](https://github.com/Fiiyinfoluwa)
+  
```

Comment:
Hey @Fiiyinfoluwa.

Remove this extra line (space), and everything looks fine to me.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Remove line 603

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix